### PR TITLE
Add "save as" to save queries into a new file

### PR DESCRIPTION
**Pull request by lucaxchaves**
Added the "save as" feature to save the current query into a new file.

The shortcuts are:

Windows: Ctrl+Shift+S
Linux: Ctrl+Shift+S
Mac OS: Cmd+Shift+S

Related issue: #566

### CODE REVIEW

File: src/renderer/actions/queries.js [hunk 1]
```diff
@@ -169,7 +176,7 @@ export function saveQuery () {
         { name: 'All Files', extensions: ['*'] },
       ];
 
-      let filename = (currentQuery.filename || await fileHandler.showSaveDialog(filters));
+      let filename = getFileName(currentQuery, isSaveAs, filters);
```
Comment (MasterOdin) [inline src/renderer/actions/queries.js:179]: Need to add an `await` here as `getFileName` returns `Promise<string>`. Without it, get and error where `filename` is a promise object and passing that to `fileHandler.saveFile` breaks.

```suggestion
      let filename = await getFileName(currentQuery, isSaveAs, filters);
```

**Review by MasterOdin — CHANGES_REQUESTED** [review]
Thanks for the change. With the suggested change, it works perfectly.

File: src/renderer/actions/queries.js (hunk 1 above)
Comment (lucaxchaves) [reply src/renderer/actions/queries.js:179]: Good catch!!

**Review by MasterOdin — APPROVED** [review]

**Comment by MasterOdin** [comment]
Thanks for contributing!

**Pull request merged**